docs/dietpi_tools.md

```diff
@@ -396,20 +408,34 @@ Run `dietpi-autostart`.
 
 ![DietPi-Autostart screenshot](assets/images/dietpi-autostart.jpg)
 
+!!! info "Autostart option in `dietpi.txt`"
+    The autostart option can also be set via the file `dietpi.txt`. See option  
```

Comment:
It's important to note that this only works before booting DietPi the first time.

Reply:
Yes, of course. I will add this.